eProsima/Fast-RTPS
# [1.9.x] Separate performance tests for Ctest [6943]

This PR separates the CTest performance tests in one for each case:
* Create `test/performance/<test_type>` sub-directories
* Add XML files to configure each test
* Re-work python scripts to take arguments so that the tests can be parameterized.
* Generate one CTest for each combination of inter/intra-process (note that VideoTest does not support intra-process yet), reliability kind, and security (only for inter-process).

On ThroughputTest:
* Add intra-process feature.
* Add a recovery_times file so that one test run can combine different payloads, demands, and recovery times.
* Generate a CSV file with all the date gathered in the run, (instead of one per payload)
* Style and other corrections

On VideoTest:
* Only compile if CMake flag `VIDEO_TESTS` is set to `ON`. This is because video test may not run on every PR, and because there are still some corrections missing.
